core/build.gradle

```diff
@@ -9,7 +9,7 @@ def projectName = "YARG (Yet another report generator)"
 def projectDescription = "Open source library for report generation. Provided by Haulmont."
 def projectUrl = "https://github.com/cuba-platform/yarg"
 
-def artifactVersion = '2.2-SNAPSHOT'
+def artifactVersion = '2.3-KLAUS'
```

Comment:
Please remove this change from the pull request. Otherwise, it will be merged into the master (it is not right)

Reply:
I've resolved the issue and reverted the artifact version.